.github/workflows/contracts.yml

```diff
@@ -25,9 +25,8 @@ jobs:
         with:
           PATTERNS: |
             **/**.rs
-            **/**.go
-            go.mod
```

Comment:
We should still have these two lines right?

Reply:
do we want to run contract tests if go changes? These aren't interacting with the chain at all

Reply:
Go tests can via a query, no?

Reply:
I don't think so. The rust tests are fetching the chain from git, so the version is specified: osmosis-testing-13.1.2 uses osmosis-13.1.1, for example. So even if "main" changes, the underlying rust test will not behave differently.

Now, this may not be what we want. We may want to be able to test the contracts against osmosis main so we are warned that we need to update them if something changes. It almost feels like we would want to test with both the version used by the contract *and* main.

@iboss-ptk Does it make sense to have a release of osmosis-testing that always points to main so we can do this?